languages/elixir/exercises/concept/chessboard/.docs/introduction.md

```diff
@@ -0,0 +1,11 @@
+## ranges
+
+Ranges represent a sequence of one or many consecutive integers. They are created by connecting two integers with `..`.
+
+```elixir
+1..5
+```
+
+Ranges can be ascending or descending. They are always inclusive of the last value.
```

Comment:
```suggestion
Ranges can be ascending or descending. They are always inclusive of the first and last values.
```

Reply:
(I'm guessing this is true? But worth being explicit either way)

Reply:
Yes!